src/igvfd/tests/indexing/test_report_tsv.py

```diff
@@ -31,3 +31,16 @@ def test_batch_download_human_donor_report_download(workbook, testapp):
     res = testapp.get('/report.tsv?type=HumanDonor&sort=accession')
     disposition = res.headers['content-disposition']
     assert disposition.startswith('attachment;filename="human_donor_report') and disposition.endswith('.tsv"')
+
+
+def test_multitype_report_download(workbook, testapp):
+
+    res = testapp.get('/multireport.tsv?institute_label=Stanford')
+    assert res.headers['content-type'] == 'text/tsv; charset=UTF-8'
+    disposition = res.headers['content-disposition']
+    assert disposition.startswith('attachment;filename="institute_label=Stanford') and disposition.endswith('.tsv"')
+    lines = res.body.splitlines()
+    assert b'/multireport/' in lines[0]
+    assert lines[1].split(b'\t') == [
+        b'ID', b'UUID', b'Title', b'Aliases', b'Awards', b'Name', b'Status', b'Principle Investigator', b'Institute Label', b'Submitted By'
+    ]
```

Comment:
Can we add some more tests here with other combinations of multiple types/abstract types?

Reply:
Sure. I can add more.